### PR TITLE
nodes rearrangement (#488)

### PR DESCRIPTION
**Pull request by DawidekZagajnik**
PR Title format: 

    nodes rearrangement (#488)

**Please** add a meaningful description for your change here

----

To make clear that you license your contribution under the MIT you have to acknowledge this by using the following check-box.

 - [X] I hereby declare this contribution to be licenced under the [MIT license](https://opensource.org/licenses/MIT)

### CODE REVIEW

**Review by atompie — COMMENTED** [review]
This solution is not used in any test. Adding it to any existing test like test_end_2_end.py would at least run it and find out if it is woking. 

Add it after

```python
        flow = Flow.build("End2End flow", id=flow_id)
        flow += debug('event') >> start('payload')
        flow += start('payload') >> increase_views('payload')
        flow += increase_views('payload') >> end('payload')
```

 You may also create a new test if the nodes do not overlap or the coordinates where changed.

File: tracardi/process_engine/action/v1/flow/start/start_action.py [hunk 1]
```diff
@@ -73,6 +73,8 @@ def register() -> Plugin:
             keywords=['start node'],
             type="startNode",
             icon='start',
+            width=200,
```
Comment (atompie) [inline tracardi/process_engine/action/v1/flow/start/start_action.py:76]: why the start node was changed with this PR?

Comment (DawidekZagajnik) [reply tracardi/process_engine/action/v1/flow/start/start_action.py:76]: This is due to the difference in shape on front-end between Start node and other nodes - had to be included.

**Comment by DawidekZagajnik** [comment]
added nodes rearranging to test, test passes

**Pull request merged**